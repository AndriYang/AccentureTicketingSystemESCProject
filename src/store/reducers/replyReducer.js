const initState = {

}

const replyReducer = (state= initState, action) => {
  switch(action.type){

    case 'REPLY_CREATED':
      console.log('created reply', action.reply);
      return state;

    case 'REPLY_ERROR':
      console.log('create reply error', action.err);
      return state;

    default:
      return state;
  }
};

export default replyReducer;
