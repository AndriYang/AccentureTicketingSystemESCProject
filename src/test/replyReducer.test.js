import replyReducer from '../store/reducers/replyReducer'

describe('With snapshots', () => {
  it('should return initial state', () =>{
    expect(replyReducer(undefined, {})).toMatchSnapshot();
  })

  it('should handle REPLY_CREATED', ()=>{
    const action = {
      type: 'REPLY_CREATED',
    }
    expect(replyReducer(undefined, {})).toMatchSnapshot();
  })

  it('should handle REPLY_ERROR', ()=>{
    const action = {
      type: 'REPLY_ERROR',
    }
    expect(replyReducer(undefined, {})).toMatchSnapshot();
  })
})
