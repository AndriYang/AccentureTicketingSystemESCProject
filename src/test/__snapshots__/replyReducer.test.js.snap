// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots should handle REPLY_CREATED 1`] = `Object {}`;

exports[`With snapshots should handle REPLY_ERROR 1`] = `Object {}`;

exports[`With snapshots should return initial state 1`] = `Object {}`;
